blamo/lib/Unit/unit.dart

```diff
@@ -148,47 +150,54 @@ class _UnitPageState extends State<UnitPage> {
                             children: <Widget>[
                               FormBuilderTextField(
                                 textInputAction: TextInputAction.next,
+                                keyboardType: TextInputType.number,
                                 focusNode: formNodes[0],
                                 attribute: 'depth-ub',
-                                validators: [FormBuilderValidators.numeric()],
+                                validators: [FormBuilderValidators.numeric(), FormBuilderValidators.max(0)],
                                 decoration: InputDecoration(labelText: "Depth Upper Bound (-m)"),
                                 initialValue: formatValue(unitToBuildFrom.depthUB.toString()),
                                 onChanged: (void nbd){updateUnitObject();},
                                 onFieldSubmitted: (v){
                                   FocusScope.of(context).requestFocus(formNodes[1]);
                                 },
                                 /*onEditingComplete: (){
-                                  debugPrint("Updating object");
-                                  unitObject.depthUB = double.parse(_fbKey.currentState.fields['depth-ub'].currentState.value);
-                                },*/
+                                debugPrint("Updating object");
+                                unitObject.depthUB = double.parse(_fbKey.currentState.fields['depth-ub'].currentState.value);
+                              },*/
                               ),
                               FormBuilderTextField(
                                 textInputAction: TextInputAction.next,
+                                keyboardType: TextInputType.number,
                                 focusNode: formNodes[1],
                                 attribute: 'depth-lb',
-                                validators: [FormBuilderValidators.numeric()],
+                                validators: [FormBuilderValidators.numeric(), FormBuilderValidators.max(0), (lower){
+                                  if(_fbKey.currentState != null && lower != null && _fbKey.currentState.fields['depth-ub'].currentState.value != null && double.tryParse(_fbKey.currentState.fields['depth-ub'].currentState.value) != null && double.tryParse(lower) != null && double.tryParse(lower) >= double.tryParse(_fbKey.currentState.fields['depth-ub'].currentState.value))
+                                    return "Depth Lower Bound must be lower than Depth Upper Bound";
+                                  return null;
+                                }],//Custom validator that checks if lower bound is lower than upper bound
                                 decoration: InputDecoration(labelText: "Depth Lower Bound (-m)"),
```

Comment:
Should we remove the negative from the unit here so it'd read "(m)?" Currently (-m) could imply that whatever the user inputs will be made negative after the fact, which isn't the case.